Scene/scene.js

```diff
@@ -74,6 +74,13 @@ class Scene {
         return id;
     }
 
+    async UpdateResolution(resolution) {
+        // get the current scene id
+        // if the id is same and the resolution is the same, do nothing
```

Comment:
We'll make the assumption that the resolution given will always be different from what is currently in the scene, so don't worry about this condition.